recipes-core/core-framework/redhawk-codegen_2.2.2.bb

```diff
@@ -18,11 +18,11 @@
 # along with this program.  If not, see http://www.gnu.org/licenses/.
 #
 require core-framework-2.2.2.inc
-inherit setuptools redhawk-sysroot
+inherit setuptools redhawk-sysroot  
```

Comment:
None of these changes are necessary.  The fix referenced in the PR can be simplified to no-exec of the configure task (replace do_configure's function with `do_configure[noexec] = "1"`).  The distutils class changed in a way that caused `setup.py` to run in a way it wasn't meant to run, hence the errors.  We don't actually need that step.